### PR TITLE
Add replacement environment variable for `workingdirinit-image`

### PR DESCRIPTION
**Pull request by nikhil-thomas**
Add a new environment variable for repalcing `workingdirinit-image` in
openshift releases.

We have a new image in tektoncd/pipeline `workingdirinit-image`. Added by: [Support workingDir init on Windows #4475](https://github.com/tektoncd/pipeline/pull/4475)
used here: https://github.com/tektoncd/pipeline/blob/3c6354c3e9ce5761725bf99499f91d002be4503b/config/controller.yaml#L74

Signed-off-by: Nikhil Thomas <nikthoma@redhat.com>

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/operator/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

```release-note
NONE
```

### CODE REVIEW

File: operatorhub/openshift/config.yaml [hunk 1]
```diff
@@ -56,6 +56,13 @@ image-substitutions:
       containerName: openshift-pipelines-operator
       envKeys:
       - IMAGE_PIPELINES_ARG__KUBECONFIG_WRITER_IMAGE
+- image: registry.redhat.io/openshift-pipelines/pipelines-workingdirinit-rhel8@
```
Comment (vdemeester) [inline operatorhub/openshift/config.yaml:59]: Could we re-use an existing image ?

Comment (nikhil-thomas) [reply operatorhub/openshift/config.yaml:59]: we can, if we decide so.
Let us merge this as this follows the default expectation.

Then if we decide to use an existing image, we can change the image name part. This entry  will server as a placeholder

Comment (vdemeester) [reply operatorhub/openshift/config.yaml:59]: sounds good 👍🏼

**Review by pradeepitm12 — APPROVED** [review]
/lgtm

**Pull request merged**